src/stylesheets/components/_sidenav.scss

```diff
@@ -1,8 +1,28 @@
 
 .usa-sidenav-list {
   @include usa-sidenav-list;
+  @include margin(null (-$site-margins-mobile) null (-$site-margins-mobile));
+  border-bottom: 1px solid;
+  border-top: 1px solid;
+
+  @include media($medium-screen) {
+    @include margin(null 0);
+  }
+
+  @include media($large-screen) {
+    border: none;
```

Comment:
This also removes left and right borders, even though there's no left and right borders present. Can we be explicit with `border-top: none` and `border-bottom: none`?

Reply:
will do